index.php

```diff
@@ -4,8 +4,10 @@
 $valet_home_path = is_dir($valet_xdg_home) ? $valet_xdg_home : $valet_old_home;
 $valet_config = json_decode(file_get_contents("$valet_home_path/config.json"));
 $tld = isset($valet_config->tld) ? $valet_config->tld : $valet_config->domain;
+$self = isset($valet_config->default) ? basename($valet_config->default) : 'valet-dashboard';
```

Comment:
`valet-dashboard` should use the directory name of the repository rather than a hardcoded string. I personally use `dashboard` so this wouldn't match in my case as-is.

Reply:
Any idea how to check for the directory name in a reliable way?

Reply:
`basename(__DIR__)` 😄

Reply:
Actually, now that I think about it, this shouldn't check `default` at all. It should only skip itself. 

If the default is a different site, it shouldn't be skipped in the list.

Reply:
In which case, this can just be inlined below:

```php
if ($site === basename(__DIR__)): continue; endif

Reply:
wooops! Nice catch, yeah makes sense, shouldn't check for default site. :)